bin/semver-tool

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
```

Comment:
Is this the same script that's in the common module? If we find a bug in it and need to fix the script, how do we keep them in sync?